code/datums/jobs.dm

```diff
@@ -1227,11 +1227,13 @@ ABSTRACT_TYPE(/datum/job/civilian)
 /datum/job/special/research_assistant
 	name = "Research Assistant"
 	linkcolor = "#9900FF"
-	limit = 1
+	limit = 2
 	wages = PAY_UNTRAINED
 	low_priority_job = 1
-	slot_jump = list(/obj/item/clothing/under/color/white)
-	slot_foot = list(/obj/item/clothing/shoes/white)
+	slot_jump = list(/obj/item/clothing/under/color/lightpurple)
```

Comment:
![image](https://user-images.githubusercontent.com/358431/154867900-af211b4c-6229-4797-b6d0-69df75eda930.png)
Idk. I don't much like such a vivid bright color on a jumpsuit used for an actual job. But maybe other people would like it :iiam: